ceph/ceph/upgrade.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+NAMESPACE=${NAMESPACE:-"ceph"}
+TIMEOUT=${TIMEOUT:-10}
+UPGRADE_ORDER="mgr mon osd mds rgw rbd-provisioner"
+KUBECTL="kubectl"
```

Comment:
declared but used only once although `kubectl` is called many times

Reply:
This is used for debugging purpose, for kubectl delete. Will replace.